s3cmd

```diff
@@ -68,6 +68,12 @@ except ImportError:
     # python2 fallback code
     from distutils.spawn import find_executable as which
 
+try:
+    # python 3 support
+    from urlparse import urlparse
+except ImportError:
+    from urllib.parse import urlparse
```

Comment:
It is a detail, but you can probably exchange the imports and have the python3 "from urlib.parse import urlparse" case first as it should now be the default case.